### DISCUSSION TITLE
How to run the code on windows

### DISCUSSION THREAD

**Discussion by xNemesys** [TinyMPC/TinyMPC#39] [Q&A]
It seems that when I build with MSVC, it complains missing dirent.h and UNIX header libraries.

Did I miss out on any README ? Thanks!

**Comment by plancherb1** [comment]
I defer to others who have compiled and run things on Windows before (I have only done Linux) but I know that some folks have gotten things to run nicely using Windows Subsystem for Linux (WSL) and you can connect that through VS Code as well (https://code.visualstudio.com/docs/remote/wsl). Maybe someone else has other methods for Windows but that might be a good place to start!
